tools/validators/instance_validator/tests/fake_instances/GOOD/translation_identical.yaml

```diff
@@ -17,24 +17,24 @@
 SDC_EXT-17-GUID:
   type: HVAC/SDC_EXT
   code: SDC_EXT-17
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
   translation:
     shade_extent_percentage_command:
       present_value: "points.shade_extent_percentage_command.present_value"
       units:
-        key: "points.shade_extent_percentage_command.units"
+        key: "pointset.points.shade_extent_percentage_command.units"
         values:
           percent: "%"
 
 SDC_EXT-18-GUID:
   type: HVAC/SDC_EXT
   code: SDC_EXT-18
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980755"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.